test/e2e/lazy_activation_policy_test.go

```diff
@@ -192,6 +193,22 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			})
 		})
 
+		// Combined Case 3 (Policy preemption)
+		// refer: https://github.com/karmada-io/karmada/blob/release-1.9/docs/proposals/scheduling/activation-preference/lazy-activation-preference.md#combined-case-3-policy-preemption
+
+		ginkgo.It("Combined Case 3 (Policy preemption)", func() {
+			ginkgo.By("step 1: deployment propagate success when policy created before it", func() {
+				waitDeploymentPresentOnCluster(originalCluster, namespace, deploymentName)
+			})
+
+			ginkgo.By("step 2: create PP2 (match nginx, cluster=member2, lazy, priority=2, preemption=true)", func() {
+				policyHigherPriority.Spec.ActivationPreference = "" // remove lazy activationPreference field
+				framework.CreatePropagationPolicy(karmadaClient, policyHigherPriority)
+				waitDeploymentPresentOnCluster(modifiedCluster, namespace, deploymentName)
+				policyHigherPriority.Spec.ActivationPreference = policyv1alpha1.LazyActivation // set lazy activationPreference, so the code below this line don't face the issues in future
```

Comment:
@chaosi-zju I was wondering if I should keep this line or there is a better way.

Reply:
you can remove this line, because the variable here doesn't affect other cases~

besides, for better, you can add another step, just like:

```go
ginkgo.By("step 3: clean up", func() {
  framework.RemovePropagationPolicyIfExist(karmadaClient, namespace, policyHigherPriorityName)
})
```